### ISSUE TITLE
bundled release crashes on win7

### ISSUE THREAD

**Issue by cosmojg** [knossos-project/knossos_cuber#1]
Running the cuber Python files from the command prompt works beautifully, and this is true for both the GUI and the command line interface. However, running knossos_cuber_gui.exe from the bundled release simply opens up a command prompt with a blinking cursor and hangs.

**Comment by Optiligence** [comment]
It takes pretty long to load here, how long did you wait? 
And what are your system specs? 
Is there no output at all?

**Comment by cosmojg** [comment]
Allowing it to run longer consistently results in the program crashing with the classic, "knossos_cuber_gui.exe has stopped working."

As for system specs, attached are the results of running DxDiag.
[DxDiag.txt](https://github.com/knossos-project/knossos_cuber/files/851827/DxDiag.txt)

**Comment by mdraw** [comment]
I just tested win64_knossos_cuber_gui.zip on Windows 10. Here is what I found:
- Startup time is also long here (~ 1 min) with oddly low CPU usage.
- Then the GUI window appears and I can click around and select input/output folders fine.
- Running a small test batch through it: It gets unresponsive in the middle of the downsampling step (I can't get more specific because the cubing log window is unresponsive too and I can't copy-paste anything or get a traceback).

I will try to make installers with https://github.com/conda/constructor. If this works better, we can publish constructor installers for all platforms.

**Comment by Optiligence** [comment]
It was built on Win10 and it would probably work on 7 and 10 if it was built on 7, I didn’t get around doing that yet tho. I also don’t understand why it breaks on 7 (somewhere within Qt), all the raw packages (python, pyqt, etc.) are compatible with 7 _and_ 10, so the only way I see it breaking is if there was some stuff generated after installation and that was packaged by pyinstaller (or even generated by it). I also didn’t investigate _that_ further.

**Comment by mdraw** [comment]
I created an installer with constructor: https://github.com/knossos-project/knossos_cuber/releases/download/v1-alpha/win64_knossos_cuber_gui_installer.exe (based on https://anaconda.org/mdraw/knossos_cuber, which you can also install directly if you already have Anaconda).
I added the build recipes I used in https://github.com/knossos-project/knossos_cuber/commit/bda9231d4d2804837b560ce649c80e8d19a47941.

The conda package and the installer work on my Linux and Windows systems. Could you try these on your systems and report back if everything works now?

**Comment by mdraw** [comment]
I am uploading an installer for Linux as well at https://github.com/knossos-project/knossos_cuber/releases/tag/v1-alpha.

Currently those installers are huge and bundle a lot of unneeded dependencies. I am sure I can significantly reduce them by explicitly excluding some packages, but before I go into detail I wanted to make sure that those installers actually *work* on all systems.

**Comment by cosmojg** [comment]
Yep, it works on Windows 7! Thank you. :) As you said, it's definitely a hefty install.

A couple questions: Is the ultimate goal to have the cuber polished and running in the same way that KNOSSOS itself is? Do you plan on eventually integrating the cuber into KNOSSOS rather than keeping it as a separate application?

**Comment by Optiligence** [comment]
To come back to my earlier point, the pyinstaller bundle also crashes on win7 even when built there.
